app/models/concerns/episode_media.rb

```diff
@@ -0,0 +1,118 @@
+require "active_support/concern"
+
+module EpisodeMedia
+  extend ActiveSupport::Concern
+
+  def complete_media
+    contents.complete_or_replaced.group_by(&:position).values.map(&:first)
```

Comment:
What is `:position`?

Reply:
The audio segments in the episode (the `Content`s) have a `position` field.  To determine what order we the segments of an episode should be sorted/played in.